baremaps-openapi/src/main/java/com/baremaps/openapi/services/ApiListingResource.java

```diff
@@ -34,14 +37,24 @@
 
   public ApiListingResource() {
     BeanConfig beanConfig = new BeanConfig();
-    beanConfig.setVersion("1.0.0");
+    beanConfig.setVersion(getVersion());
     beanConfig.setSchemes(new String[] {"http"});
     beanConfig.setBasePath("/");
     beanConfig.setResourcePackage("com.baremaps.openapi.services");
     beanConfig.setScan(true);
     this.swagger = beanConfig.getSwagger();
   }
 
+  public String getVersion() {
+    try {
+      Properties properties = new Properties();
+      properties.load(Resources.getResource("version.txt").openStream());
```

Comment:
Sonar is complaining about not using try-with-resources. I can't blame sonar :sweat_smile: